src/ClientApi/Actions/SendTransfer/Action.php

```diff
@@ -532,9 +532,7 @@ protected function prepareTransfers()
             }
 
             // remove the checksum
-            $transfer->setRecipientAddress(
```

Comment:
`removeChecksum` returns a new instance of the address without a checksum.

Reply:
Okay, you are right. I've fixed it, but I think we should refactor it and don't remove checksum at this point.